CTF/README.md

```diff
@@ -0,0 +1,24 @@
+# CTF challenge
+
+We have an exciting challenge to get you folks tinkering
+with rust. We've provided three encrypted files: encrypted[1..3].txt. Your job
+is to figure out the encryption scheme for each of those and implement the functions
+crack1() and crack2() under `tinker.rs`. Quick hint: Two of the files are encrypted
+with the same encryption method but different keys.
+
+## Setup
+Build the project with `cargo build`. This will generate the executable file
+/target/debug/tinker (tinker.exe on Windows). The executable describes how
+to run itself, try `tinker --help`.
+
+## Challenge
+Implement the two functions `crack1()` and `crack2()`. The first function should return
+a `Result<i32, dyn Error>`, where the Ok value is the integer key.
```

Comment:
Should we provide an `Error` type that we expect? Error types might be another form of hint that we can provide?

Reply:
I like that idea. Let me see what I can do